money/money.go

```diff
@@ -29,13 +31,25 @@ type Money struct {
 }
 
 // NewMoney returns a new instance of Money.
-// The money is a representation of myst in a uint64 form, with the decimal part expanded.
-// This means, that one myst is equivalent to 10 0000 000.
-func NewMoney(amount *big.Int, currency Currency) Money {
-	return Money{amount, currency}
+// Expected `amount` value for 1 myst is equal to 1_000_000_000_000_000_000.
+// It also allows for an optional currency value to be passed,
+// if one is not passed, default config value is used.
+func NewMoney(amount *big.Int, currency ...Currency) Money {
```

Comment:
Since the package is `money`, may I suggest simply naming it `money.New()`?

Reply:
Of couse. It was like this and I didn't think about changing it. Nice.